backends/s3/client.go

```diff
@@ -75,6 +82,26 @@ func (c *client) Backup() error {
 	return c.putObject(bytes.NewReader(data), c.Bucket, backupKey)
 }
 
+func (c *client) IsWritable() bool {
+	return c.writable
+}
+
+func (c *client) SetWritable(writable bool) error {
+	c.writable = writable
+	// TODO: Implement me!
+	return builtinerrors.New("not implemented")
+}
+
+func (c *client) Lock() error {
+	// TODO: Implement me!
```

Comment:
I need to come up with a solution for s3 before merging this, otherwise it'll break for me.

Reply:
.. not to say it shouldn't be implemented asap —`s3` support is still dangerous to use.

Reply:
Agreed, tracking in https://github.com/bndw/pick/issues/174